src/zoid/buttons/util.js

```diff
@@ -389,10 +399,18 @@ export const getModal: () => Object = memoize(() => {
       return new Promise((resolve) => {
         const script = document.createElement("script");
         script.src = modalBundleUrl();
+        script.setAttribute("data-namespace", namespace);
```

Comment:
I believe `modal.js` only uses `data-pp-namespace` so is this safe to remove?

Reply:
It seems that in practice data-namespace works and data-pp-namespace does not. data-pp-namespace fails to make a request to smart/modal

Reply:
this was discussed and fixed externally; for posterity, `data-pp-namespace` is the correct attribute.